client-tez/pom.xml

```diff
@@ -26,61 +27,81 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <groupId>org.apache.uniffle</groupId>
     <artifactId>rss-client-tez</artifactId>
     <version>0.8.0-SNAPSHOT</version>
     <packaging>jar</packaging>
     <name>Apache Uniffle Client (Tez)</name>
 
     <dependencies>
         <dependency>
-            <groupId>org.apache.uniffle</groupId>
-            <artifactId>rss-client</artifactId>
+            <groupId>org.apache.tez</groupId>
```

Comment:
Could we remove the `version`? The pom will extend the parent tez version. We use the parent pom to manage the third party package version.

Reply:
ok, we removed it and pushed it.